packages/material/scss/multiselect/_layout.scss

```diff
@@ -13,21 +13,27 @@
             }
 
             & + .k-button {
-                margin: ($input-padding-y / 2) 0 0 ($input-padding-y / 2);
+                margin: ($input-padding-y / 2) ($input-padding-y / 2) 0 0;
```

Comment:
This rule simply duplicates the above one. If that's the fix, you need to remove the rule.